.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "cli/bin/vendor/python-future"]
 	path = cli/bin/vendor/python-future
 	url = https://github.com/PythonCharmers/python-future.git
+[submodule "cli/bin/vendor/pythonfutures"]
+	path = cli/bin/vendor/pythonfutures
+	url = https://github.com/agronholm/pythonfutures.git
```

Comment:
the `futures` module is already being added, so there's no need to add it again. This was done as part of #524